src/components/ha-sidebar.ts

```diff
@@ -744,12 +745,12 @@ class HaSidebar extends LitElement {
         width: 100%;
       }
 
-      paper-listbox::-webkit-scrollbar {
+      .ha-scrollbar::-webkit-scrollbar {
```

Comment:
Why not use `haStyleScrollbar` here?

Reply:
Because I forgot to go back :) and update it after copying